R/plumber.R

```diff
@@ -1061,6 +1032,9 @@ plumber <- R6Class(
     notFoundHandler = NULL,
     maxSize = NULL, # Max request size in bytes
 
+    apiHandler = NULL,
```

Comment:
```suggestion
    apiHandler = identity,
```

Reply:
setting it to something else than NULL makes it locked : cannot change value of locked binding for 'apiHandler', will change in init method